test/units/testsuite-75.sh

```diff
@@ -212,6 +212,11 @@ resolvectl log-level debug
 
 # Start monitoring queries
 systemd-run -u resmontest.service -p Type=notify resolvectl monitor
+# Wait for the monitoring service to become active
+for _ in {0..9}; do
```

Comment:
There really should be something like `systemctl wait` there...

Reply:
Yeah... thankfully this hunk has been dropped a couple of commits later.